test/integration/testdata/config.yaml.tmpl

```diff
@@ -22,3 +22,5 @@ data:
     mixerAddress: istio-mixer.{{.IstioNamespace}}:9091
 {{end}}
     authPolicy: {{ .Auth }}
+    disablePolicyChecks: false
+    enableTracing: true
```

Comment:
Are these necessary to changing the discovery/refresh duration?

Reply:
Eh no.. sorry, I just wanted to include the default values to make sure that these are being parsed by Pilot.

Reply:
ah, right - I forgot what the defaults were here